### PR TITLE
add lzip support to Unarchiver.ts

### LINKED ISSUE #283: add `.lz` support for source packages
ref: https://github.com/pkgxdev/pantry/pull/4945

### PR DESCRIPTION
**Pull request by jhheider**
closes #283

### CODE REVIEW

File: lib/utils/Unarchiver.ts [hunk 1]
```diff
@@ -76,6 +80,13 @@ export class TarballUnarchiver extends Unarchiver {
         constraint
       })
       break
+    case ".tlz":
+    case ".tar.lz":
+      rv.push({
+        project: "nongnu.org/lzip",
```
Comment (mxcl) [inline lib/utils/Unarchiver.ts]: unused, sorry. I should delete the code already.

Comment (mxcl) [reply lib/utils/Unarchiver.ts]: You need to add it to the shebang for `./bin/cmd/build` instead

Comment (jhheider) [reply lib/utils/Unarchiver.ts]: even better

File: bin/cmd/build [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env -S pkgx +rsync +git +bash +gum +gh +curl +bzip2 +xz +unzip deno run --ext=ts --allow-env --allow-read --allow-write --allow-run --allow-net
+#!/usr/bin/env -S pkgx +rsync +git +bash +gum +gh +curl +bzip2 +xz +unzip +lzip deno run --ext=ts --allow-env --allow-read --allow-write --allow-run --allow-net
```
Comment (mxcl) [inline bin/cmd/build:1]: we are become epic

**Review by mxcl — APPROVED** [review]

**Comment by jhheider** [comment]
you can ignore the unit-test failure; addressed in #283

**Pull request merged**